test/__snapshots__/preset-mini.test.ts.snap

```diff
@@ -71,9 +74,10 @@ exports[`preset-mini > targets 1`] = `
 .border-b{border-bottom-width:1px;border-style:solid;}
 .border-t-2{border-top-width:2px;border-style:solid;}
 .border-x{border-left-width:1px;border-right-width:1px;border-style:solid;}
-.border-\\\\[\\\\#124\\\\]{--un-border-opacity:1;border-color:rgba(17,34,68,var(--un-border-opacity));}
+.border-\\\\[\\\\#124\\\\]{border-color:#124;}
```

Comment:
When used without the opacity, we should keep the variable so it can be controlled by other utils.